test/assets/schedule_get_extract_refresh_tasks.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<tsResponse xmlns="http://tableau.com/api" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://tableau.com/api http://tableau.com/api/ts-api-2.3.xsd">
+  <extracts>
+    <extract id="task1"
+      priority="1"
+      type="incremental-or-full" >
```

Comment:
The type doesn't matter here, but to make it useful for other tests could you put the value "IncrementalRefresh" instead of "incremental-or-full" here?